src/components/groups/group.middlewares.ts

```diff
@@ -131,10 +133,28 @@ export const validateGroup = () => {
     check('description', 'A leírás max 500 karakter lehet')
       .optional({ nullable: true })
       .isString()
-      .isLength({ max: 500 })
+      .isLength({ max: 500 }),
+    check('maxAttendees', 'Legalább 1, maximum 100 fő vehet részt!')
+      .optional({ checkFalsy: true })
+      .isInt({ min: 1, max: 100 })
   ]
 }
 
+export const checkMaxAttendees = asyncWrapper(
+  async (req: Request, res: Response, next: NextFunction) => {
+    const users: User[] = await Group.relatedQuery('users').for(req.params.id)
+    if (users.length > (req.body.maxAttendees || 100)) {
```

Comment:
We do not have to query all the related users, we need the number of them, for which we could use:
```typescript
const userCount = await Group.relatedQuery('users').for(req.params.id).count()
```

This way, the query will be more goal-oriented and efficient.